app/package.json

```diff
@@ -2,13 +2,21 @@
   "name": "@klimadao/app",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 3001",
+    "dev": "npm run lingui:compile && next dev -p 3001",
     "build": "next build",
     "start": "next start",
     "export": "next export",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lingui:extract": "NODE_ENV=test lingui extract",
```

Comment:
why `test`?

Reply:
There seem to be a bug with `lingui extract` when running it in production mode in particular setups. This is a workaround
https://github.com/lingui/js-lingui/issues/433